runtime/src/lib.rs

```diff
@@ -253,6 +254,7 @@ impl_opaque_keys! {
 		pub grandpa: Grandpa,
 		pub babe: Babe,
 		pub im_online: ImOnline,
+		pub authority_discovery: AuthorityDiscovery,
```

Comment:
should probably be in the line below so the existing key types don't switch their location.